<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>852</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_login_msg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>lb_login_msg</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>911</width>
      <height>531</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>거래량 / 전고점 돌파</string>
     </attribute>
     <widget class="QTableWidget" name="tw_top_trade_stock">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>402</width>
        <height>445</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>시간</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>주가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>상승율</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>코드</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pb_top_trade_stock">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>START / STOP</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_top_trade_stock_telegram">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram On/Off</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_top_trade_stock_cycle">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>110</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>주기(초)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>실시간 상승</string>
     </attribute>
     <widget class="QPushButton" name="pb_real_time_stock">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>상승 종목</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>130</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>분봉</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_real_time_stock">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>702</width>
        <height>445</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>시간</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>상승율</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>주가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>현재 주가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>최고 상승률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>코드</string>
       </property>
      </column>
     </widget>
     <widget class="QCheckBox" name="cb_real_time_stock_telegram">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telegram</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_real_time_stock_volume">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>160</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>160</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>거래량(%)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_le_real_time_stock_cycle">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>100</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>이평선 포함</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_real_time_stock_rate">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>190</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>상승율(%)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_real_time_stock_cycle">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>130</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pb_real_time_cost">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>230</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>실시간 주가 감시</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>729</x>
        <y>310</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>고점 하락율(%)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_real_time_cost_rate_1">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>310</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_real_time_cost_rate_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>370</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>729</x>
        <y>370</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>시점 하락율(%)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_high_cost_1">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>280</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>고점 하락율 </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_high_cost_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>340</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>시점 하락율 </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>실시간 하락</string>
     </attribute>
     <widget class="QTableWidget" name="tw_real_time_stock_low">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>702</width>
        <height>445</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>시간</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>상승율</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>주가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>현재 주가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>최고 상승률</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>코드</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="te_system_msg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
